Makefile

```diff
@@ -76,7 +76,7 @@ builder:
 	IMAGE_TAG=latest make -C infrastructure/docker/builder build
 
 kuberpult:
-	make -C pkg/api all
```

Comment:
I think we need both of them. The frontend-service has a sim-link to the `pkg/api` directory.